stories/panel.stories.js

```diff
@@ -58,4 +58,44 @@ storiesOf('Panel', module)
         })
       );
     })
+  )
+  .add(
+    'with collapsed',
+    withHits(({ search, container, instantsearch }) => {
+      search.addWidget(
+        instantsearch.widgets.panel({
```

Comment:
Prefer the usage of a variable for the `panel`. We can consider Storybook part of our documentation. It might be hard to read at first for users that never saw this kind of pattern.